package.json

```diff
@@ -16,16 +19,21 @@
     "jest": "^29.4.1",
     "ts-jest": "^29.0.5",
     "tslib": "^2.5.0",
-    "typescript": "4.2.3"
+    "typescript": "^5.0.4"
   },
   "dependencies": {
     "axios": "^1.3.2",
+    "https-browserify": "^1.0.0",
     "stellar-sdk": "^10.4.1",
-    "ts-loader": "8.0.7",
-    "webpack": "4.41.5",
-    "webpack-cli": "3.3.10"
+    "stream-http": "^3.2.0",
+    "ts-loader": "^9.4.2",
+    "url": "^0.11.0",
+    "util": "^0.12.5",
+    "webpack": "^5.83.1",
+    "webpack-cli": "^5.1.1"
   },
   "scripts": {
+    "prepack": "yarn build",
```

Comment:
this script will be needed when we publish to npm; `prepare` tells npm what script to run to pack everything up before publishing to npm when you run `npm publish`

Reply:
@piyalbasu just double checking should it be `prepack` or `prepare` as you mentioned?